scripts/install/install_unix.sh

```diff
@@ -13,165 +13,638 @@
-            desired_state="$(dpkg -l observiq-otel-collector | tail -n1 | cut -b 1)"
-            if [ "$desired_state" = "i" ]; then
-                echo "Existing DEB installation detected, uninstalling..."
-                dpkg -r "observiq-otel-collector"
-            else
-                # The package is already uninstalled;
-                # This case can be hit when the package was uninstalled, but config files are left behind,
-                # which dpkg still keeps track of.
-                echo "dpkg was detected, but no observiq-otel-collector package is currently installed, skipping..."
-            fi
-        else
-            echo "dpkg was detected, but no observiq-otel-collector package is currently installed, skipping..."
-        fi
-    fi
+extract_package()
+{
+  case "$package_type" in
+    deb)
+      dpkg -i "$out_file_path" > /dev/null
+      ;;
+    rpm)
+      rpm -U "$out_file_path" > /dev/null
+      ;;
+    *)
+      error "Unrecognized package type"
+      return 1
+      ;;
+  esac
+  return 0
+}
 
-    if command -v rpm > /dev/null 2>&1; then
-        if rpm -q observiq-otel-collector > /dev/null 2>&1; then
-            echo "Existing RPM installation detected, uninstalling..."
-            rpm -e "observiq-otel-collector"
-        else
-            echo "rpm was detected, but no observiq-otel-collector package is currently installed, skipping..."
-        fi
-    fi
+# This will display the results of an installation
+display_results()
+{
+    banner 'Information'
+    increase_indent
+    info "Collector Home:     $(fg_cyan "/opt/observiq-otel-collector")$(reset)"
+    info "Collector Config:   $(fg_cyan "/opt/observiq-otel-collector/config.yaml")$(reset)"
+    info "Start Command:      $(fg_cyan "sudo systemctl start observiq-otel-collector")$(reset)"
+    info "Stop Command:       $(fg_cyan "sudo systemctl stop observiq-otel-collector")$(reset)"
+    info "Logs Command:       $(fg_cyan "sudo journalctl -u observiq-otel-collector.service")$(reset)"
+    decrease_indent
+
+    banner 'Support'
+    increase_indent
+    info "For more information on configuring the collector, see the docs:"
+    increase_indent
+    info "$(fg_cyan "https://github.com/observiq/observiq-otel-collector/tree/main#observiq-opentelemetry-collector")$(reset)"
+    decrease_indent
+    info "If you have any other questions please contact us at $(fg_cyan support@observiq.com)$(reset)"
+    increase_indent
+    decrease_indent
+    decrease_indent
+
+    banner "$(fg_green Installation Complete!)"
+    return 0
 }
 
-main() {
-    # Determine system parameters
-    set_arch
-    set_package_type
-    set_file_name
+uninstall_package()
+{
+  case "$package_type" in
+    deb)
+      dpkg -r "observiq-otel-collector" > /dev/null 2>&1
+      ;;
+    rpm)
+      rpm -e "observiq-otel-collector" > /dev/null 2>&1
+      ;;
+    *)
+      error "Unrecognized package type"
+      return 1
+      ;;
+  esac
+  return 0
+}
```

Comment:
We don't check if anything is actually present to uninstall. Which might error out at this step.

ALSO, if nothing is installed, and we do not error out here, the user has no feedback that there was nothing to uninstall. Which might be fine, but I recall this was feedback that was given for the first version of the uninstall script, so I thought I might add it here.

Reply:
So yeah I made the choice to redirect stderr and stdout to dev null so this no-ops if nothing exists. I think this is probably a product question on the behavior. I'll ask.

Reply:
Spoke with Joe H. he's ok with this being a no-op. Resolving this for now.